utility/src/main/java/org/oppia/android/util/statusbar/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+"""
+Utility to change the color of Status Bar.
```

Comment:
Suggest instead: ``Utilities for managing the Android status bar.`` (so that we aren't overconstraining what can go in this package)

Reply:
done